Robus/src/robus.c

```diff
@@ -98,6 +106,8 @@ void Robus_Init(memory_stats_t *memory_stats)
  ******************************************************************************/
 void Robus_Loop(void)
 {
+    // network timeout management
+    Robus_RunNetworkTimeout();
```

Comment:
What is it?

Reply:
Ok I get it, this is a function to retrieve a node waiting for a failed detection. In a failed detection configuration we should receive an assert from ID 1.
This assert is an event, which is more practical than this polling function.

Reply:
Ok but we don't want the node to be in ASSERT state when a detection failed. That's why I choose to implement this in polling mode.